docs/docs/queries/query-types.md

```diff
@@ -1,204 +1,486 @@
 # Query Types
 
-Your **Query Type** determines how the output of your dataview query looks like. LIST, TABLE and CALENDAR always operates on page level, whereas TASK queries operate on `file.tasks` level.
-
-## List Queries
-
-Lists are the simplest view, and simply render a list of pages (or custom fields) which match the query.
-To obtain a list of pages matching the query, simply use:
-
-=== "Syntax"
-    ```
-    LIST FROM <source>
-    ```
-=== "Query"
-    ``` sql
-    LIST FROM #games/mobas OR #games/crpg
-    ```
-=== "Output"
+The **Query Type** determines how the output of your dataview query looks like. It is the **first and only mandatory** specification you give to a dataview query. There are four available: `LIST`, `TABLE`, `TASK` and `CALENDAR`.
+
+The Query Type also determine on which **information level** a query is executed.  `LIST`, `TABLE` and `CALENDAR` operate on **page level**, whereas `TASK` queries operate on `file.tasks` level. More on that on the `TASK` Query Type. 
+
+You can combine **every Query Type with all available [Data Commands](data-commands.md)** to refine your result set. Read more about the interconnection between Query Types and Data Commands on [How to Use Dataview](../index.md#how-to-use-dataview) and the [structure page](structure.md).
+
+!!! summary "Query Type"
+    The Query Type determines the output format of a query. It's the only mandatory information for a query.
+
+## LIST
+
+`LIST` queries output a bullet point list consisting out of your file links or the group name, if you decided to [group](data-commands.md#group-by). You can specify up to **one additional information** to output alongside with your file or group information.
```

Comment:
I did not want to write additional field because that's only half the story; you can output a literal (with little sense) or a calculation, too, that perhaps contains multiple field values, so "one additional field" isn't sufficient.

Reply:
Let's merge as is and think about it later.